docs/dev.md

```diff
@@ -0,0 +1,34 @@
+# Developer guide
+
+### Table of Contents  
+[Global architecture](#global-archi)  
```

Comment:
[Info] In english global means relative to the whole world. Here you rather want to say "complete" or "overall" ?